makefile

```diff
@@ -28,6 +28,14 @@ integration-test: swagger
 
 ci-test: test integration-test
 
+# Build a (local) Docker image for Caldera
+# https://caldera.readthedocs.io/en/latest/Installing-Caldera.html#docker-deployment
+# We must select `master` as the branch; 5.0.0 contains a critical bug: https://github.com/mitre/caldera/issues/2949
```

Comment:
Another less than ideal thing: we cannot pin the Caldera version, because the most recent version (5.0.0) we have to use (because of the UI) contains a critical bug.